handlers/main.yml

```diff
@@ -5,9 +5,9 @@
   service:
     name: "{{ fluentd_service_name }}"
     state: restarted
-  notify: Wait 3 seconds (restart)
+  notify: Wait 5 seconds (restart)
 
-- name: Wait 3 seconds (restart)
+- name: Wait 5 seconds (restart)
```

Comment:
the comment below must be updated.
`# XXX listed order of these handlers does matter. "Wait 3 seconds (restart)"`

Reply:
fixed in 338363d